addons/mail/static/src/views/web/fields/emojis_text_field/emojis_text_field.xml

```diff
@@ -5,6 +5,9 @@
         <xpath expr="//span[1]" position="attributes">
             <attribute name="t-ref">targetReadonlyElement</attribute>
         </xpath>
+        <xpath expr="//*[hasclass('o_field_input_buttons')]" position="attributes">
+            <attribute name="class" add="emoji"  separator=" "/>           
+        </xpath>
```

Comment:
If you apply my comment then you can also remove this one.

Reply:
You're right won't be useful anymore.